scripts/timing.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+from collections import namedtuple
+import datetime
+import functools
+import os
+import random
+import string
+from subprocess import call
+
+RESULTS_BLOCKLTR = '''  ____                 _ _
+ |  _ \ ___  ___ _   _| | |_ ___ _
+ | |_) / _ \/ __| | | | | __/ __(_)
+ |  _ <  __/\__ \ |_| | | |_\__ \_
+ |_| \_\___||___/\__,_|_|\__|___(_)'''
+
+BLORGLY_BACKEND_DIR = os.path.join(os.environ['GOPATH'], 'src/github.com/windmilleng/blorgly-backend')
```

Comment:
Should we be using blorg-backend for our benchmarks since @yuindustries discovered that blorgly-backend is kind of borked?

Reply:
Blorgly-Backend is not actually hooked up to the frontend. The original idea was for it to provide random strings, but it neither generates those strings nor seems to have ever been successfully tested communicating with the frontend when everyone's deployed to K8s. (It was tested on local, though, and I assume it works there.)

Reply:
hmm so i feel like as long as it deploys (which it does), it doesn't matter if it works -- but also i don't have strong feelings and am happy to defer to others here

Reply:
I think it's fine to use blorg-backend for now, since for milestone purposes I'd want anything related to output to be merged in sooner rather than later. And I agree with your logic. 

I myself ended up using blorg-backend for the Docker for Mac test (PITA though it is, with its Cockroach DB setup and such) because I couldn't figure out another way of verifying that "everything works."